t_stress/test_ddos.py

```diff
@@ -3,158 +3,206 @@
 __license__ = "GPL2"
 
 import os
+import time
 import unittest
 
 from helpers import dmesg, remote, tf_cfg
 from test_suite import sysnet, tester
 
+TEMPESTA_IP = tf_cfg.cfg.get("Tempesta", "ip")
 DURATION = int(tf_cfg.cfg.get("General", "duration"))
 MHDDOS_DIR = os.path.join("tools/mhddos")
-PROXY_PATH = f"{MHDDOS_DIR}/files/proxies/http.txt"
+PROXY_PATH = f"{MHDDOS_DIR}/files/proxies.txt"
 MHDDOS_PATH = f"{MHDDOS_DIR}/start.py"
 THREADS = int(tf_cfg.cfg.get("General", "stress_threads"))
 CONNS = int(tf_cfg.cfg.get("General", "concurrent_connections"))
+RPC = int(tf_cfg.cfg.get("General", "stress_requests_count"))
 
 
-# @unittest.skipIf(
-#     TEMPESTA_IP.startswith("127."), "Please don't use loopback interface for this test."
-# )
+@unittest.skipIf(
+    TEMPESTA_IP.startswith("127."), "Please don't use loopback interface for this test."
+)
 class TestDDoSL7(tester.TempestaTest):
     clients = [
         {
             "id": f"mhddos",
             "type": "external",
             "binary": "python3",
             "cmd_args": "",
-        }
+        },
+        {"id": "curl", "type": "curl", "http2": False, "ssl": True},
     ]
 
     tempesta = {
-        "config": """
+        "config": f"""
+listen 80 proto=http;
 listen 443 proto=h2,https;
 
 cache 2;
```

Comment:
May be check how we stay under ddos without cache also?

Reply:
I added description to test for these cases. I was not able responses from upstream without frang.